gateway-package/src/main/resources/bin/start.sh

```diff
@@ -140,6 +175,50 @@ ADD_OPENS="--add-opens=java.base/java.lang=ALL-UNNAMED
         --add-opens=java.base/sun.nio.ch=ALL-UNNAMED
         --add-opens=java.base/java.io=ALL-UNNAMED"
 
+get_enabled_protocol_limit()
+{
+    target=$1
+    type=$2
+    key_component=ZWE_configs_zowe_network_${target}_tls_${type}Tls
+    key_zowe=ZWE_zowe_network_${target}_tls_${type}Tls
+    echo ${!key_component:-${!key_zowe:-}}
+}
+
+get_enabled_protocol()
+{
+    target=$1
+    enabled_protocols_min=$(get_enabled_protocol_limit "${target}" "min")
+    enabled_protocols_max=$(get_enabled_protocol_limit "${target}" "max")
+
+    if [ "${enabled_protocols_min:-}" == "${enabled_protocols_max:-}" ]; then
+        echo "${enabled_protocols_max:-}"
+    elif [ -z "${enabled_protocols_min:-}" ]; then
+        echo "${enabled_protocols_max:-}"
+    else
+        enabled_protocols_max=${enabled_protocols_max:-TLSv1.3}
+
+        enabled_protocols=,SSLv3,TLSv1,TLSv1.1,TLSv1.2,TLSv1.3,TLSv1.4,
+        enabled_protocols=${enabled_protocols%%,${enabled_protocols_max}*}
+        enabled_protocols=${enabled_protocols#*${enabled_protocols_min},}
+        if [ ! -z ${enabled_protocols} ]; then
+          enabled_protocols=",${enabled_protocols}"
+        fi
+        echo "${enabled_protocols_min}${enabled_protocols},${enabled_protocols_max}"
+    fi
+}
```

Comment:
Are these going to be carried over to the rest of the java components?

Reply:
I believe these should be applied to all the start.sh scripts, yes